### DISCUSSION TITLE
Add inert variant for [[inert]]

### DISCUSSION THREAD

**Discussion by codehz** [tailwindlabs/tailwindcss#14123] [Ideas]
https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/inert
I think it is useful attribute... 

it is like disabled state, but more "powerful"

example use case: `inert:opacity-0 opacity-100 transition-opacity`

**Comment by RobinMalfait** [comment]
Hey!

Good idea, added a PR for this: https://github.com/tailwindlabs/tailwindcss/pull/14129
